.github/workflows/ci.yml

```diff
@@ -0,0 +1,137 @@
+name: CI
+
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+
+jobs:
+  docs:
+    name: "Documentation"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Build Docs
+        run: |
+            pip install -e .[dev,docs]
+      - name: Test docbuilding
+        run: |
+            invoke docs --clean --build
+      # TODO: Upload the docs to some place
+      # where they can be served. Potentially build
+      # this in a different CI tool that can serve
+      # html artifacts
```

Comment:
No need; we use RTD

Reply:
Yes, and I would keep RTD for the docs for each version on `master`. I was thinking that it might be nice to also do this for each PR individually.

We are already building the docs as a test and it might be nice to also upload them somewhere, so we can inspect the formatting in the actual PR before merging it with `master`.

Reply:
actually, RTD seems to offers this feature: https://docs.readthedocs.io/en/stable/pull-requests.html

I don't think we need to add it in this PR, but we may want to keep the comment there for the future. If we decide to add it this feature in a new PR, I'd like to open an issue about it so that it is in the backlog.

Reply:
Sounds good!